pkg/frontend/mysql_cmd_executor.go

```diff
@@ -358,6 +360,117 @@ func handleShowColumns(ses *Session) error {
 	return nil
 }
 
+func getDateFromLocalBatch(obj interface{}, bat *batch.Batch) error {
```

Comment:
Here too, where explain need all this?   Maybe I missed something?

Please ask Mo Chen to take a look at the explain, esp, analyze part.

Reply:
OK, you are right. I have confirmed to Mo Chen that these are really not needed. I use another way to achieve them.